src/Client/Pages/BuildingBlock/SearchComponent.fs

```diff
@@ -179,14 +185,15 @@ let private AddBuildingBlockButton (model: Model) dispatch =
 [<ReactComponent>]
 let Main (model: Model) dispatch =
     let state_bb, setState_bb = React.useState(BuildingBlockUIState.init)
+    let inputValue, setInputValue = React.useState ""
```

Comment:
please put this into `BuildingBlock.Model` with a more descriptive name

Reply:
we want to share the same information between page switches. Everything in the ReactComponent is lost when switching to another page

Reply:
to clarify: "page" means sidebar tab